relude.cabal

```diff
@@ -72,6 +72,8 @@ library
                        Relude
                            Relude.Applicative
                            Relude.Base
+                           Relude.Bifoldable
+                           Relude.Bitraversable
```

Comment:
I think it's better to move the reexports into `Relude.Foldable.Reexport` module. This module already reexports some parts of `Data.Foldable` and `Data.Traversable`. And `relude` already has quite a lot of modules, so I would like to avoid creating new modules unless necessary... Having single better word for `Foldable` and `Traversable` together is also a nice option 👍 But I can't think of one...

Reply:
:ok_hand: